kubernetes/core-pod.yml

```diff
@@ -1,30 +1,30 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: busybox-pod
+  name: core-pod
   labels:
-    app: busybox
+    app: core-app
 spec:
   containers:
-  - name: busybox-container
-    image: busybox
+  - name: core-app-container
+    image: jmatejka/happy-bank-core:1.0.0
     args:
     - /bin/sh
     - -c
-    - touch tmp/healthy; sleep infinity;
+    - touch healthy; ls; sleep infinity;
     ports:
      - containerPort: 80
```

Comment:
What does the number of the container pod mean? What is it for? Is it correct for our core app microservice?

Reply:
Thank you! As per documentation flask application uses port 5000 by default. Now it works!